shared/src/main/scala/com/cibo/evilplot/colors/Color.scala

```diff
@@ -147,9 +148,9 @@ object HEX {
 }
 
 object Color {
-  private implicit val cfg = com.cibo.evilplot.JSONUtils.minifyProperties
-  implicit val encoder: Encoder[Color] = io.circe.generic.extras.semiauto.deriveEncoder[Color]
-  implicit val decoder: Decoder[Color] = io.circe.generic.extras.semiauto.deriveDecoder[Color]
+  private implicit val cfg: Configuration = com.cibo.evilplot.JSONUtils.minifyProperties
+  implicit val encoder: Encoder[Color] = io.circe.generic.extras.semiauto.deriveConfiguredEncoder[Color]
+  implicit val decoder: Decoder[Color] = io.circe.generic.extras.semiauto.deriveConfiguredDecoder[Color]
```

Comment:
Some usages of `deriveEncoder` and `deriveDecoder` were deprecated.  This PR addresses those deprecation warnings by migrating to `deriveConfiguredEncoder` and `deriveConfiguredDecoder`